app/build.gradle

```diff
@@ -217,7 +217,7 @@ dependencies {
     implementation "com.jakewharton.rxbinding:rxbinding-support-v4:$rx_binding_version"
     implementation "com.jakewharton.timber:timber:5.0.1"
     implementation 'com.optimizely.ab:android-sdk:3.13.2'
-    implementation 'com.stripe:stripe-android:20.11.0'
+    implementation 'com.stripe:stripe-android:20.5.0'
```

Comment:
Why reverting the stripe's library version?

Reply:
I will change it